src/Option_Command.php

```diff
@@ -218,9 +237,10 @@ public function list_( $args, $assoc_args ) {
 		global $wpdb;
 		$pattern = '%';
 		$exclude = '';
-		$fields = array( 'option_name', 'option_value' );
+		$fields = array( 'option_id', 'option_name', 'option_value' );
```

Comment:
You should just add the ID here, that is a breaking change. People who are already using this command will not expect the format to suddenly change from one version ot the next.

If you need the ID for your tests, you can adapt the tests to specifically add the `option_id` to the list of displayed fields instead.